redaxo/src/addons/structure/lib/service_slice.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/**
+ * @package redaxo\structure
```

Comment:
Die Klasse gehört ins content-Plugin, das Mutter-Addon kennt noch gar keine Slices.

Reply:
> das Mutter-Addon kennt noch gar keine Slices.

Das bedeutet dann das die ganze Klasse Quatsch ist?

Reply:
Sie müsste in addons/structure/plugins/content/lib liegen